content/notifications/explanation/recipient-lookup/_index.nb.md

```
@@ -1,45 +1,45 @@
 | -------------- | ------------------------------------------------------------------------------------------------------------- |
-| status         | The result of the initial lookup.                                                                             |
-| isReserved     | A list of national identity numbers for recipients who have chosen to opt out of electronic communication.    |
-| missingContact | A list of national identity and/or organization numbers for recipients who do not have contact details.       |
+| status         | Resultatet av det innledende oppslaget.                                                                             |
+| isReserved     | En liste over fødselsnumre for mottakere som har valgt å reservere seg mot elektronisk kommunikasjon.    |
+| missingContact | En liste over fødselsnumre og/eller organisasjonsnumre for mottakere som ikke har kontaktdetaljer.       |
 
-The status property can have one of three values:
-| Status         | Description                                     |
+Status-egenskapen kan ha en av tre verdier:
+| Status         | Beskrivelse                                     |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)